pronouncing/__init__.py

```diff
@@ -217,8 +218,14 @@ def rhymes(word):
     :returns: a list of rhyming words
     """
     phones = phones_for_word(word)
-    if len(phones) > 0:
-        return [w for w in rhyme_lookup.get(rhyming_part(phones[0]), [])
-                if w != word]
+    combined_rhymes = []
+    if phones:
+        for element in phones:
+            combined_rhymes.append([w for w in rhyme_lookup.get(rhyming_part(
+                                    element), []) if w != word])
+        combined_rhymes = list(chain.from_iterable(combined_rhymes))
+        combined_rhymes.sort()
+        unique_combined_rhymes = set(combined_rhymes)
```

Comment:
iirc, I'm not sure if using set/list to remove duplicates will produce the desired results across all the versions of python that we're targeting—probably best to sort the list post-dedup.